apps/prairielearn/src/lib/question.js

```diff
@@ -2053,4 +2053,17 @@ module.exports = {
       );
     });
   },
+
+  /**
+   * Expose the renderer in use to the client so that we can easily see
+   * which renderer was used for a given request.
+   *
+   * @param {import('express').Response} res
+   */
+  setRendererHeader(res) {
+    const renderer = res.locals.question_renderer;
+    if (renderer) {
+      res.set('X-PrairieLearn-Question-Renderer', renderer);
+    }
+  },
```

Comment:
I felt like this would be useful information to have to verify whether a given request is flowing through the new renderer or not? We should be able to tell this without question from the state of the feature flag, but still...

Reply:
Can't hurt!